### DISCUSSION TITLE
HTTP status code 403

### DISCUSSION THREAD

**Discussion by Thelion78** [rinigus/pure-maps#608] [General]
Installed to Xperia 10 ii on Vanha Rauma 4.4.0.64 the v 3.1.0 from Chum. Choosing MapTiler as map gives HTTP status code 403. Is it something wrong I have, or is it just not working? OpenSlopeMap gives just white screen, HSL and OpenTopoMap works.

**Comment by rinigus** [comment]
It is the end of the month - monthly quota for MapTiler has been used up. I recommend to register for some of the supported services and configure to access those.
